test/functional/plugin/lsp_spec.lua

```diff
@@ -813,3 +813,35 @@ describe('LSP', function()
     end)
   end)
 end)
+
+describe('LSP', function()
```

Comment:
could you move your tests inside the upper `describe('LSP', function()` line 740. 

Feel free to reference the spec to explain the order, i.e., add an url

Reply:
done